staging/src/k8s.io/genericapiserver/pkg/authentication/user/doc.go

```diff
@@ -16,4 +16,4 @@ limitations under the License.
 
 // Package user contains utilities for dealing with simple user exchange in the auth
 // packages. The user.Info interface defines an interface for exchanging that info.
-package user // import "k8s.io/kubernetes/pkg/auth/user"
+package user // import "k8s.io/genericapiserver/pkg/authentication/user"
```

Comment:
I'm fine with k8s.io/apiserver.  I think we have until 1.6 dcut to change our mind, so I'll update tomorrow.